blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc

```diff
@@ -0,0 +1,28 @@
+
+= JBoss Fuse Tooling - Support of Global configurations
+
+:page-date:2016-05-20
+:page-layout: blog
+:page-author: apupier
+:page-tags: [jbtis,fuse]
+
+Note: This article has been posted also on http://developers.redhat.com/blog/2016/05/18/jboss-fuse-tooling-support-of-global-configurations/[Red Hat Developers Program blog]
+
+A new editor tab is now available to manipulate global configurations.
+

```

Comment:
I added the second part only.
From the link to "earlier post", it explains how to install it.